packages/cozy-stack-client/src/CozyStackClient.spec.js

```diff
@@ -489,7 +489,7 @@ describe('CozyStackClient', () => {
       )
     })
 
-    it(`should NOT try to refresh the current token when receiving a 'forbidden' response with 'invalid token' error in 'www-authenticate' (not expired token)`, async () => {
+    it(`should try to refresh the current token when receiving a 'forbidden' response with 'invalid token' error in 'www-authenticate' (not expired token)`, async () => {
```

Comment:
It seemed to me that the test had to be reverted with the conditional value change in stack-client